DOC-2278: Combine Login and Password Policies; add Password Policy rules to Configuration Parameters (4.1) (#513)
tigergraph/server-docs: modules/security/pages/login-password.adoc

= Complexity password policy and Login protection
:description: Login, Password
:pp: {plus}{plus}
:page-aliases: README.adoc, readme.adoc

> Tigergraph provides complexity password policy and login protection to improve the security.

Reviewer: ```suggestion
TigerGraph provides a complex password policy and login protection to enhance security.
```

Reviewer 2: @anjaliIthapeTG We don't have a complex policy (or at least I hope we don't). We offer optional built-in rules for the complexity of passwords, e.g. the password must have a letters, numbers, and special characters.

Reviewer: @victorleeTG My apologies for the oversight. I’ll review the documentation mentioned on the website and will update again

> Tigergraph provides complexity password policy and login protection to improve the security.

Reviewer: We could also include reasons why a complex password policy and login protection are necessary and how can it help users

== Password Policy

The password policy is a set of rules and requirements that dictate how passwords should be created, password expiration times, and password rotation rules to ensure security.
See xref:tigergraph-server:security:password-policy[] for a detailed description

== Login Protection

Login protection is a security feature that helps safeguard user accounts from unauthorized access, enhancing overall account security.

> See xref:tigergraph-server:security:login-protection[] for a detailed description

Reviewer 2: ```suggestion
See xref:login-protection[] for a detailed description
```

Reviewer 2: If referencing a page within the same component/repo, do not give the name of the component.
The reason why is: referencing the component with jump to the newest version of that component (if you didn't specify a version).
